c-cannons/bangazon-initial_site
# Delete product

## Pull Request
#### Descriptions

1. Added Delete Product functionality.
1. Products can only be deleted if they are not part of an order.
1. If a product that is part of an order gets a delete command, the exception is handled.


#### Steps to Test

You must provide clear steps for any teammate to test the code.

1. `git fetch origin delete-products`
1. `git checkout delete-products`
1. Navigate to My Products
1. Products that are part of an order will not have a Delete link
1. Products that CAN be deleted will have a Delete link
1. When deleted, a flash message will appear 'Product Deleted"

#### Link to Feature Ticket

https://github.com/c-cannons/bangazon-initial_site/issues/19
